packages/modules/order/src/services/order-module-service.ts

```diff
@@ -2238,8 +2238,8 @@ export default class OrderModuleService<
       return {
         action: ChangeActionType.RETURN_ITEM,
         internal_note: item.internal_note,
-        reference: data.reference,
-        reference_id: shippingMethodId,
+        reference: data.reference ?? "fulfillment",
+        reference_id: data.reference_id ?? shippingMethodId,
```

Comment:
This reference is generic, and could be related to any "reference" and its ID.
We haven't yet a "return" table. But I believe this will be referenced there.